app/jobs/destroy_old_ressources_job.rb

```diff
@@ -38,7 +38,17 @@ def destroy_inactive_users # rubocop:disable Metrics/MethodLength
   end
 
   def destroy_useless_rdvs
-    useless_rdvs = Rdv.where.missing(:participations).where("rdvs.created_at < ?", date_limit).destroy_all
+    useless_rdvs = Rdv.where.missing(:participations).where("rdvs.created_at < ?", date_limit)
+
+    # On envoit pas de webhook de destroy rgpd pour les rdvs du département 13
+    bdr_rdv_ids = useless_rdvs.joins(organisation: :department)
+                              .where(departments: { number: "13" })
+                              .pluck(:id)
+    useless_rdvs.each do |rdv|
+      rdv.should_send_webhook = bdr_rdv_ids.exclude?(rdv.id)
+    end
```

Comment:
gros nitpick: Je trouve ça un poil dur à lire cognitivement parlant, pour plus de clarté on pourrait faire quelque chose du genre
```suggestion
    useless_rdvs.each do |rdv|
      next unless rdv.id.in?(bdr_rdv_ids)
      
      rdv.should_send_webhook = false
   end
```

Reply:
Tu as raison :)